.github/workflows/ros-ci.yaml

```diff
@@ -0,0 +1,64 @@
+name: ros-ci
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  asan:
+    strategy:
+      fail-fast: false
+      matrix:
+        env:
+          - ROS_DISTRO: rolling
+          - ROS_DISTRO: humble
```

Comment:
If we wanted to expand this matrix to include multiple build types or both static/shared libs, would that be feasible?

Reply:
Yes, we could even make the mixin a variable here.